### ISSUE TITLE
Make an organism selection directive

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1558]
I thought this was in a directive, but it isn't.  We should extract it so it can be reused:

```html
 <select name="select-organism" ng-model="data.selectedOrganism"
          ng-options="o as o.full_name for o in organisms">
    <option value="">Select an organism ...</option>
 </select>
```
(from genotype_gene_list.html)

**Comment by jseager7** [comment]
The plan is to extract the picker into a directive, allow the picker to fetch its own list of organisms, filter the list (by host or pathogen, etc) based on some criteria, and to allow containing directives to bind to the picker, primarily so they can read the `selectedOrganism` value and react accordingly.

**Comment by kimrutherford** [comment]
I've just pushed some fixes.  The genotype manage page is now a directive.

Probably best to git pull before you do to much.

**Comment by jseager7** [comment]
Thanks for letting me know. I'm up to date now.

**Comment by jseager7** [comment]
Closing this now, since it's been implemented by #1563 and I've tested that it can integrate with the meta-genotype page. Factoring this out has probably caused other bugs with the genotype page that we'll need to fix, though.
